renderer/screens/flips/components/flip-pics.js

```diff
@@ -14,6 +14,11 @@ import theme from '../../../shared/theme'
 import {convertToBase64Url} from '../utils/use-data-url'
 import {IMAGE_SEARCH_PICK, IMAGE_SEARCH_TOGGLE} from '../../../../main/channels'
 import {IconButton} from '../../../shared/components/button'
+import {getImageURLFromClipboard} from '../../../shared/utils/clipboard'
+
+const mousetrap = require('mousetrap')
+
+const isMac = process.platform === 'darwin'
```

Comment:
this won't work. you don't have access to electron process here. need to pass `global.isMac` through the `preload.js`, like that

Reply:
<img width="210" alt="image" src="https://user-images.githubusercontent.com/50833934/68019158-d3dc6b80-fcbc-11e9-9b6d-57d6acea71d5.png">
here from mac ^

Reply:
or just _Cmd/Ctrl+V_ 🙂